.build-tools/README.md

```diff
@@ -8,7 +8,7 @@ The CLI is written in Go and based on the [cobra](https://github.com/spf13/cobra
 
 You have two ways to run the CLI:
 
-1. Within the `build-tools` directly, you can run the CLI with `go run .` directly. For example `go run . help` shows the help page.
+1. Within the `build-tools` directly, you can run the CLI with `go run .` directly. For example `go run . help` shows the help page. Make sure that `GOOS` and `GOARCH` are set to the correct values for your system.
```

Comment:
In fact, those values are set autosomally, so a better fix is that remove GOOS set in Makefile. Is it possible?

Reply:
This is for manually running only. Makefile sets `GOOS` for e2e tests today, which is required. If you are relying on Makefile, it should handle GOOS as well as build-tools compilations automatically.

Reply:
But the GOOS in Makefile only works during Make, unless users set it by hand, or it is always same as system.

Reply:
I miswrote, Makefile does not set `GOOS`, it looks for it, and uses `TARGET_OS_LOCAL` (`shell uname`) if it's not present.